### PR TITLE
Generate cleaned-tbl format data

### PR DESCRIPTION
**Pull request by nikhil96sher**

### CODE REVIEW

File: experiments.md [hunk 1]
```diff
@@ -16,7 +16,20 @@ export DATA_DIR=/absolute/path/to/data  # where you want to put scale=100/partit
 docker run --rm \
     -v ${DATA_DIR}:/dataset/tpch:rw \
     --name dataset deepola-wake:sigmod2023-data \
-    bash data-gen.sh 100 100 /dataset/tpch
+    bash data-gen.sh 100 100 /dataset/tpch && \
+    python3 convert-to-parquet.py /dataset/tpch/scale=100/partition=100/tbl
```
Comment (nikhil96sher) [inline experiments.md:20]: @mIXs222 Do you know of a way to run these two commands together? Seems like the `&&` is separating out the `docker run` and `python3 convert-to-parquet` as two separate commands and thus giving error.

Comment (mIXs222) [reply experiments.md]: I think you can put them in quote, like `"sleep 3 && echo hello"`

File: experiments.md [hunk 2]
```diff
@@ -44,9 +44,26 @@ docker run --rm \
...
+docker run --rm \
+    -v ${DATA_DIR}:/dataset/tpch:rw \
+    --name dataset deepola-data:sigmod2023 \
+    python3 convert-to-parquet.py /dataset/tpch/scale=${SCALE}/partition=100/tbl
 ```
 
-TODO: Automatically generate cleaned-parquet formats as well (as part of data-gen.sh)
+#### Generate Dataset for ProgressiveDB
+- Note: Ensure that the data has been generated already. This script only converts `lineitem` table to `cleaned-tbl` format.
+- For TPC-H (Scale 100, Partition 100)
```
Comment (mIXs222) [inline experiments.md:56]: I forgot this but could you replace 100 with SCALE here and below?

File: scripts/plot_tpch.py [hunk 3]
```diff
@@ -35,6 +49,8 @@ def wake_read_time(scale, partition, num_runs, pdx):
     all_results = [
         ("WAKE-final", wake_read_time(args.scale, args.partition, args.num_runs, -1)),
         ("WAKE-first", wake_read_time(args.scale, args.partition, args.num_runs, 1)),
+        ("Postgres", postgres_read_time(args.scale)),
```
Comment (mIXs222) [inline scripts/plot_tpch.py:52]: Move these up so they're on the left like in the paper?

**Review by mIXs222 — APPROVED** [review]

**Pull request merged**